Source/Android/app/src/main/res/layout/list_item_skylander_slot.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout android:id="@+id/root"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:paddingVertical="@dimen/spacing_medlarge">
+
+    <Button
+        android:id="@+id/button_clear_skylander"
```

Comment:
Could you swap the position of this button with the create button? Create/Load/Clear looks better to me, what do you think?